torchtune/datasets/_alpaca.py

```diff
@@ -6,16 +6,17 @@
 
 from functools import partial
 
-from torchtune.data import AlpacaInstructTemplate
-from torchtune.datasets._instruct import InstructDataset
+from torchtune.datasets._instruct import instruct_dataset, InstructDataset
 from torchtune.modules.tokenizers import Tokenizer
 
 
 def alpaca_dataset(
     tokenizer: Tokenizer,
+    *,
```

Comment:
nit: be consistent here. In other things you put the kw delineator as the first arg, and in others you put it after tokenizer. Just pick one, either works.

Reply:
Hm for instruct and chat dataset builders there are many more required positional arguments which is why I put the asterisk in the beginning. I can just put it in the beginning for all of them